recipes/bottle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bottle" %}
+{% set version = "0.12.9" %}
+{% set sha256 = "45285ad084ca054d821ceef8bd95462efd38e13ecbef13a82c22b6472a6f4b2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bottlepy/bottle/archive/{{ version  }}.tar.gz
```

Comment:
Is it better to use pypi @jakirkham? What is the idea?

Where are all these "suggested guidelines" being collected?